### DISCUSSION TITLE
各位觉得这个面板如何？

### DISCUSSION THREAD

**Discussion by wxgl** [share121/inter-knot#3409] [其他]
![Screenshot_20240821_141305](https://github.com/user-attachments/assets/3c6c51e2-0cc9-4f06-991f-2882a65ba6d3)
<img src="https://bbs-static.miyoushe.com/static/2024/03/21/a63b1fdb4c2a4562c7d291b230978f0a_944582820103199731.png" alt="丽都漫步-莱卡恩" width="75" >

**Comment by Legendary-agent** [comment]
挺好的，我没没有和安比合作多少，不清楚她的实力特点

**Comment by mmm1314520** [comment]
666

**Comment by mmm1314520** [comment]
666
